ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/package.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.validator
+
+import com.daml.ledger.participant.state.kvutils.Bytes
+import com.daml.lf.data.Time.Timestamp
+
+package object preexecution {
+
+  /**
+    * A raw key-value pair with a flag indicating whether the key is a log entry key or not.
+    */
+  type AnnotatedRawKeyValuePairs = Seq[(AnnotatedRawKey, Bytes)]
```

Comment:
You always have a single log entry so keeping that key-value pair separate from the rest or noting its key should suffice and simplify the consuming code.

Reply:
Good point, done.